src/applications/personalization/dashboard-2/selectors.js

```diff
@@ -1,8 +1,15 @@
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 
-export const selectShowDashboard2 = state =>
-  toggleValues(state)[FEATURE_FLAG_NAMES.dashboardShowDashboard2];
+export const selectShowDashboard2 = state => {
+  const LSDashboardVersion = localStorage.getItem('DASHBOARD_VERSION');
```

Comment:
It somehow feels wrong for a selector to rely on localStorage. I think in principle selectors should only rely on the state they are given and nothing else.

Reply:
If showing the dashboard 2 is dependent on both a feature toggle and local storage, then I think it would make sense if the selector relied on both. At the end of the day we just want a single function that gives us back the same result cross-team for whether we are showing dashboard 2